### ISSUE TITLE
[auction details] auction has 'Auction claiming' status even if funds were claimed

### ISSUE THREAD

**Issue by tatyanakurinskaya** [gnosis/ido-ux#626]
1. open app
2. start an auction
3. open auction and made some orders
4. wait for an end and .settleAuction on contract 
5. open auction and claim funds

https://ido-ux.dev.gnosisdev.com/#/auction?auctionId=93&chainId=4#topAnchor

Result: 
Auction claiming
![image](https://user-images.githubusercontent.com/863519/119844795-697bc380-bf11-11eb-8f02-cb17a2fd869c.png)
Exp.result: Auction is settled

**Comment by elena-zh** [comment]
Hi @josojo , @cmagan , 
we do not have a 'special' status for cases when an auction is ended and a user has no orders/has already claimed funds. 'Auction Claiming' is still displayed for these cases. 
I have updated the issue description and propose to create a new status for this cases. 
What do you think about it?
BTW, we have a pending issue #293 related to creation an intermediate status 'Waiting for settlement'. I think, it would be nice to add it in the near sprints.. =)

Thanks

**Comment by josojo** [comment]
Yes, we can think about creating a new status. Though I think its fine to always call it "auction claiming", even if it is just claiming for the users that actually used the auction.

Yes, good point. #293 is much more important

**Comment by elena-zh** [comment]
Thanks, Alex!
Then, I'm changing this issue priority to low as it is not so important.
